amberdata/src/endpoint/balance.ts

```diff
@@ -3,7 +3,7 @@ import { Config } from '@chainlink/types'
 import { Requester } from '@chainlink/external-adapter'
 import { isChainType, isCoinType, BLOCKCHAINS } from '.'
 
-export const Name = 'balance'
+export const Names = ['balance']
```

Comment:
I really like this concept for backwards compatibility.

What it also allows for is if we embrace that "external-adapters" are a simple wrapper that map to a data providers API we could have the naming fit the API.

For example `1forge` has an endpoint baseURL/`convert` that we use to get price data. So there we could set the name = ['price', 'convert']

Only tweak I would make is capitalizing it to denote a static property.
```suggestion
export const NAME = ['balance']
```

Reply:
I am happy that you like it. I left the naming as I saw in some places, since we use NAME for the whole adapter package(btw, we can re-use a similar logic for routing between different adapter packages, e.g by using "source", as mentioned at the Composable Adapters document). Reverting it to NAME now until we think more regarding these.